.github/workflows/Runic.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: julia-actions/cache@v2
       - name: Install Runic
         run: |
-          julia --color=yes -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl")'
+          julia --color=yes -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl", rev = "3931a4ffe3e46220e72829de367a9366de1684d2")'
       - name: Run Runic
         run: |
-          julia --color=yes -m Runic --check --diff $(git ls-files -- '*.jl')
+          git ls-files -z -- '*.jl' | xargs -0 julia --project=@runic -m Runic --check --diff
```

Comment:
I updated the README in Runic to this which should be safer against e.g. spaces in file names etc.